artifactory_test.go

```diff
@@ -5014,6 +5014,56 @@ func TestConfigAddOverwrite(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestConfigEncryption(t *testing.T) {
+	initArtifactoryTest(t, "")
+
+	// Create temp jfrog home
+	cleanUpJfrogHome, err := coretests.SetJfrogHome()
+	if err != nil {
+		return
+	}
```

Comment:
Shouldn't we assert this error?
The same applies to the `TestConfigEncryptionMissingKey` function.